tests/fft_test.py

```diff
@@ -164,18 +164,19 @@ def testFftEmpty(self):
     self.assertArraysEqual(jnp.zeros((0,), jnp.complex64), out)
 
   @parameterized.named_parameters(jtu.cases_from_list(
-      {"testcase_name": "_inverse={}_real={}_hermitian={}_shape={}_axis={}".format(
-          inverse, real, hermitian, jtu.format_shape_dtype_string(shape, dtype), axis),
+      {"testcase_name": "_inverse={}_real={}_hermitian={}_shape={}n={}_axis={}".format(
+          inverse, real, hermitian, jtu.format_shape_dtype_string(shape, dtype), n, axis),
        "axis": axis, "shape": shape, "dtype": dtype, "inverse": inverse, "real": real,
-       "hermitian": hermitian}
+       "hermitian": hermitian, "n": n}
       for inverse in [False, True]
       for real in [False, True]
       for hermitian in [False, True]
       for dtype in (real_dtypes if (real and not inverse) or (hermitian and inverse)
                                 else all_dtypes)
       for shape in [(10,)]
+      for n in [None, 1, 20]
```

Comment:
I would suggest includes some odd shapes in the tests, given that some of the padding depends on whether the input has odd or even size.